src/tree/containers/ContainersTreeItem.ts

```diff
@@ -124,6 +124,10 @@ export class ContainersTreeItem extends LocalRootTreeItemBase<DockerContainer, C
 export const NonComposeGroupName = localize('vscode-docker.tree.containers.otherContainers', 'Other Containers');
```

Comment:
TODO: should this change?

Reply:
Standalone Containers?

Reply:
@BigMorty What do you think? This string is the header for the grouping of non-compose containers.

Reply:
We talked and decided on "Individual Containers" at least for now.